scripts/vcs_wrapper

```diff
@@ -15,6 +15,11 @@ for file in $files; do
   file_args+=( "$file" )
 done
 
+filelists="$WAKE_IP_RESOURCE_FILELIST"
```

Comment:
```suggestion
filelists="$WAKE_IP_RESOURCE_FILELISTS"
```

Can you make this environment variable plural as to match the other environment variables?